modules/organization-admin/versions.tf

```diff
@@ -0,0 +1,11 @@
+terraform {
+  required_version = ">= 0.15"
+
+  required_providers {
+    aws = {
+      source                = "hashicorp/aws"
+      version               = ">= 3.0"
+      configuration_aliases = [aws.macie_administrator]
```

Comment:
The configuration_aliases are not needed, since we don't actually need multiple providers.